package.json

```diff
@@ -5,17 +5,18 @@
   "scripts": {
     "start": "webpack-dev-server --config test/screenshot/webpack.config.js --content-base test/screenshot",
     "stop": "./test/screenshot/stop.sh",
-    "capture": "cross-env MDC_COMMIT_HASH=$(git rev-parse --short HEAD) MDC_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD) mocha --compilers js:babel-core/register --ui tdd --timeout 30000 test/screenshot/capture-suite.js",
```

Comment:
wait i dont think you want to remove this cross-env part...im surprised travis passes actually

Reply:
Well linux and macOSx cmd line are the same commands. That is why it is passing. Its more like Windows would not pass.